cellprofiler/modules/exporttocellh5.py

```diff
@@ -306,21 +310,19 @@ def handle_interaction(self, master_file, pid, path):
         master_dict = self.get_dictionary().setdefault(master_file, {})
         if pid not in master_dict:
             md_head, md_tail = os.path.splitext(master_file)
-            subfile = "%s_%s%s" % (md_head, str(pid), md_tail)
```

Comment:
If there's no metadata for a category, I think I'm going to display a setting that lets the user type in a custom metadata entry. I'll default to "1" for site, "A01" for well and "Plate" for plate.